script_library/pre-flight-checks.sh

```diff
@@ -106,7 +106,7 @@ check_ansible_requirements (){
     fi
     # We need ansible version 2.8 minimum
     if [[ $(ansible --version | awk 'NR==1 { gsub(/[.]/,""); print substr($2,0,2); }' ) -lt "28" ]]; then
-        echo "Insufficent version of ansible: 2.8 or greater is required."
+        echo "Insufficent version of ansible: 2.8.3 or greater is required."
```

Comment:
Why is 2.8.3 or greater required? Don't you wan't to fix the check if it fails for your 2.8.1?